.github/workflows/publish.yaml

```diff
@@ -21,8 +21,6 @@ jobs:
           python-version: 3.x
       - name: Install Python dependencies
         run: pip install -r requirements.txt
-      - name: Install PlantUML
-        run: sudo apt-get -y install plantuml
```

Comment:
plantuml was required for the small diagram on the home page. I got rid of it in favor of drawio images created with [vscode-drawio](https://marketplace.visualstudio.com/items?itemName=hediet.vscode-drawio)

Reply:
omg, that is awesome ! 

not for us to use, but just as info since you might find it interesting:
There is  https://github.com/yuzutech/kroki, that lets you upload text and returns the rendering for plantuml and nearly everything else that does text to diagram/drawing

Reply:
FYI There is also a plugin vor IntelliJ: https://plugins.jetbrains.com/plugin/15635-diagrams-net-integration 👏